README.md

```diff
@@ -0,0 +1,77 @@
+# JavaScript Capstone Restuarant APP using API
+
+> This is a Restuarant App Developed in Vanilla JavaScript. The main objective for this project is to learn about basic JavaScript concepts, functions, arrays and objects DOM manipulation and connect to API's.
+
+### Screenshots
+
+|Main Project page
+
+|![Fastantistic Restaurant](https://user-images.githubusercontent.com/30318155/129350424-7d1d5391-4d25-44d0-94a0-b08659b2d95a.png)|
+
+## Built With
+
+- HTML5
+- CSS3
+- Bootstrap
+- Font Awesome
+- JavaScript-
+- MDB API
+- Involvement API
+- Jest Test
+
+## Our Project
+
+This project is found on the following Repo [Fantastic Restaurant](https://github.com/tnyandoro/JavaScript-Capstone/)
```

Comment:
- [ ] In the recording kindly ensure you both **appear** in the video. As required please use the zoom recording feature then upload the video to loom. Check the [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md#:~:text=You%20all%20should,with%20your%20peers.).

Reply:
Video resolved